src/templates/Footer/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import '../../i18n';
```

Comment:
this should not be imported here. Please import i18n in `main.tsx` as i18n should start initialization in root of our application